common/grpc/client.go

```diff
@@ -166,6 +171,11 @@ func loadTLSCredentials(cc ClientConfig) (credentials.TransportCredentials, erro
 		RootCAs:    certPool,
 		ServerName: cc.TLSServerName,
 	}
+
+	if cc.TLSSkipVerify {
+		config.InsecureSkipVerify = true
+	}
+
```

Comment:
You could use it directly in the `config` variable, I guess.
```suggestion
        InsecureSkipVerify: cc.TLSSkipVerify,
	}
```